client/src/components/InterviewDialog.js

```diff
@@ -0,0 +1,90 @@
+import React, { useContext } from 'react';
```

Comment:
Is this file being used anywhere?

Reply:
Yep! The useContext was used in a later commit when we create the interview since we needed to pass state.user's id for the POST request

Reply:
I meant InterviewDialog.js itself. If so, can we match the exported function name to the file name and modify the default aria text?